public/pages/workflow_detail/agentic_search/test_flow/test_flow.tsx

```diff
@@ -122,6 +125,13 @@ export function TestFlow(props: TestFlowProps) {
       .unwrap()
       .then((response) => {
         setSearchResponse(response);
+
+        // persist a new memory ID to be optionally injected into the query from the user.
+        const respMemoryId = response?.ext?.memory_id ?? '';
```

Comment:
we can store `response?.ext?.memory_id` in a separate variable and for respMemoryId we can check the type of it if it's string then have it like `respRaw.trim() : ''` in ternary operation

Reply:
sure can add. in general this shouldn't happen if the APIs don't have breaking changes, but can add.